pkg/spec/postgresql.go

```diff
@@ -107,6 +108,12 @@ func parseTime(s string) (t time.Time, wd time.Weekday, wdProvided bool, err err
 		}
 		timeLayout = "Mon:15:04"
 		wdProvided = true
+		weekday, ok := weekdays[parts[0]]
+		if !ok {
+			err = fmt.Errorf("incorrect weekday")
+			return
+		}
+		wd = time.Weekday(weekday)
```

Comment:
because time.Parse does not actually parses it, it just checks for the syntax.
Extract from the [doc](https://golang.org/pkg/time/#Parse):

> The day of the week is checked for syntax but it is otherwise ignored.